resources/js/profile-page/badges.tsx

```diff
@@ -21,11 +22,16 @@ export default class Badges extends React.PureComponent<Props> {
     return (
       <div className={classWithModifiers('profile-badges', this.props.modifiers)}>
         {this.props.badges.map((badge) => {
+          const hasDate = present(badge.awarded_at);
+          const htmlTitle = hasDate ?`<div>${badge.description}</div>
+            <div class='tooltip-default__date'>${moment(badge.awarded_at).format('LL')}</div>` : null;
```

Comment:
element class (`tooltip-default__date`) can't be used when the block (`tooltip-default`) isn't in the same file/component.

Reply:
I'm not entirely sure what you mean, as far as I'm aware I just did what `TimeWithTooltip` did.
TimeWithTooltip:
![image](https://github.com/ppy/osu-web/assets/26160742/6c56b9a2-6efb-411a-88b5-323eb361796e)
This PR:
![image](https://github.com/ppy/osu-web/assets/26160742/eeb29b4c-5881-490b-81f8-33d23547912f)

Reply:
the other one is used together in the same file where the tooltip-default is set.

Reply:
Oh okay, am I supposed to make a new class then?

Reply:
We use [BEM](https://getbem.com/naming/) conventions for css naming; the profile badge component uses `profile-badge` as its corresponding block name.